docs/githubconnector/helm-installation-tutorial.md

```diff
@@ -1,77 +1,100 @@
-Installation in Kyma using Helm
-===
+# Installing in Kyma using Helm
```

Comment:
I am not sure this doc belongs to this repository - it contains a description of general Helm usage with TLS and registering services in Kyma, despite it being a good idea to keep it written somewhere I think you should move it to some kind of internal repository.

Reply:
Good point I will move it to chart README.md as Tomek Papiernik suggested to Karol 8 days ago.

https://sap-cx.slack.com/archives/C9W38QL6B/p1564550803000100?thread_ts=1564392387.001100&cid=C9W38QL6B